### DISCUSSION TITLE
Add Sensor Hottest Core (HotSpot)

### DISCUSSION THREAD

**Discussion by Trigger911** [Rem0o/FanControl.Releases#1619] [Ideas]
Hello,

Thanks for the great software and the time you put into this project. Is there a way to add or support the hottest core or package into the sensor array? I hope I did my due diligence searching but I didn't see anyone ask this anywhere in the discussions.

**Comment by kh805794** [comment]
You can achieve this by using the HWInfo plugin. Follow the instructions [here](https://github.com/Rem0o/FanControl.HWInfo) to install and configure.

I have it set up to pull the GPU hotspot from HWInfo into FanControl. You'll need to have HWInfo open prior to running FanControl for the plugin to work.

Hope this helps!
